src/app/[entity]/[repo]/page.tsx

```diff
@@ -57,83 +62,146 @@ export default function RepoPage({
+            </DropdownMenuContent>
+          </DropdownMenu>
+
           <div className="flex gap-2">
-            <Button
-              className="h-7 !border-[#383B42] bg-[#22262C] text-xs"
-              variant="outline"
-            >
-              <Eye className="mr-2 h-4 w-4" /> Unwatch{" "}
-              <Badge className="ml-2">148</Badge>
-            </Button>
-            <Button
-              className="h-7 !border-[#383B42] bg-[#22262C] text-xs"
-              variant="outline"
-            >
-              <Zap className="mr-2 h-4 w-4" /> Zaps{" "}
-              <Badge className="ml-2">1337</Badge>
-            </Button>
-            <Button
-              className="h-7 !border-[#383B42] bg-[#22262C] text-xs"
-              variant="outline"
-            >
-              <Globe2 className="mr-2 h-4 w-4" /> Relays{" "}
-              <Badge className="ml-2">8</Badge>
-            </Button>
-            <Button
-              className="h-7 !border-[#383B42] bg-[#22262C] text-xs"
-              variant="outline"
-            >
-              <GitFork className="mr-2 h-4 w-4" /> Fork{" "}
-              <Badge className="ml-2">209</Badge>
-            </Button>
-            <Button
-              className="h-7 !border-[#383B42] bg-[#22262C] text-xs"
-              variant="outline"
-            >
-              <Star className="mr-2 h-4 w-4 text-yellow-500" /> Starred{" "}
-              <Badge className="ml-2">7k</Badge>
-            </Button>
+            <div className="hidden lg:flex lg:flex-row">
+              <Button
+                className="h-8 !border-[#383B42] bg-[#22262C] text-xs"
+                variant="outline"
+              >
+                <Eye className="mr-2 h-4 w-4" /> Unwatch
+                <Badge className="ml-2">148</Badge>
+              </Button>
+              <Button
+                className="h-8 !border-[#383B42] bg-[#22262C] text-xs"
+                variant="outline"
+              >
+                <Zap className="mr-2 h-4 w-4" /> Zaps
+                <Badge className="ml-2">1337</Badge>
+              </Button>
+              <Button
+                className="h-8 !border-[#383B42] bg-[#22262C] text-xs"
+                variant="outline"
+              >
+                <Globe2 className="mr-2 h-4 w-4" /> Relays
+                <Badge className="ml-2">8</Badge>
+              </Button>
+              <Button
+                className="h-8 !border-[#383B42] bg-[#22262C] text-xs"
+                variant="outline"
+              >
+                <GitFork className="mr-2 h-4 w-4" /> Fork
+                <Badge className="ml-2">209</Badge>
+              </Button>
+              <Button
+                className="h-8 !border-[#383B42] bg-[#22262C] text-xs"
+                variant="outline"
+              >
+                <Star className="mr-2 h-4 w-4 text-yellow-500" /> Starred
+                <Badge className="ml-2">7k</Badge>
+              </Button>
+            </div>
```

Comment:
we can factorize this in another pr i think

Reply:
yea